src/components/SignupForm/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+import logo from '../../app/assets/logo.png';
+import ButtonForm from '../ButtonForm';
+import InputText from '../InputText';
+import { isPasswordEqual, emailPattern } from '../../utils/formValidations';
+
+import { AUTH_INPUTS } from './fields';
+import styles from './styles.module.scss';
+
+function SignupForm() {
+  const { register, handleSubmit, errors, getValues } = useForm();
+
+  const onSubmit = handleSubmit(({ firstName, lastName, email, password, confirmPassword }) => {
+    const user = {
+      firstName,
+      lastName,
+      email,
+      password,
+      confirmPassword
+    };
+    console.log({ user });
+  });
+
+  return (
+    <div className={styles.signupForm}>
+      <img src={logo} className={styles.logo} />
+      <form onSubmit={onSubmit}>
+        <InputText
+          label="Nombre"
+          type="text"
+          name={AUTH_INPUTS.firstName}
+          inputRef={register({ required: 'Requerido' })}
+          error={errors[AUTH_INPUTS.firstName]?.message}
+        />
+        <InputText
+          label="Apellido"
+          type="text"
+          name={AUTH_INPUTS.lastName}
+          inputRef={register({ required: 'Requerido' })}
+          error={errors[AUTH_INPUTS.lastName]?.message}
+        />
+        <InputText
+          label="Email"
+          type="text"
+          name={AUTH_INPUTS.email}
+          inputRef={register({
+            required: 'Requerido',
+            pattern: emailPattern
+          })}
+          error={errors[AUTH_INPUTS.email]?.message}
+        />
+        <InputText
+          label="Password"
+          type="password"
+          name={AUTH_INPUTS.password}
+          inputRef={register({ required: 'Requerido' })}
+          error={errors[AUTH_INPUTS.password]?.message}
+        />
+        <InputText
+          label="Confirmar password"
+          type="password"
+          name={AUTH_INPUTS.confirmPassword}
+          inputRef={register({
+            required: 'Requerido',
+            validate: confirmPassword => {
+              if (!isPasswordEqual(confirmPassword, getValues('password'))) {
+                return 'Las contraseñas no coinciden';
+              }
+              return true;
```

Comment:
are you sure you need to return this?

Reply:
according to the linter config I need to return something in an arrow function, otherwise it returns an error when doing the commit